react-components/src/util/I18NWrapper.js

```diff
@@ -16,8 +16,14 @@ export default function withI18N(WrappedComponent, intlData) {
     }
 }
 
-export function getMessage(id) {
+export function getMessage(id, options) {
+    if (options && options.values) {
```

Comment:
You might be able to use destructuring here, and if `FormattedMessage` doesn't care about null/undefined `values`, then you wouldn't need the `if` check:
```jsx
export function getMessage(id, { values }) {
     return (
         <FormattedMessage id={id} values={values} />
     );
 }
```

Reply:
Oh wait, I don't think this will work if the 2nd argument is null/undefined. Nevermind 😊